src/main/java/uk/gov/justice/digital/delius/controller/secure/StaffResource.java

```diff
@@ -79,4 +84,18 @@ public StaffDetails getStaffDetailsForUsername(
         log.info("getStaffDetailsList called with {}", usernames);
         return staffService.getStaffDetailsByUsernames(usernames);
     }
+
+    @ApiOperation(value = "EXPERIMENTAL: Return list of of currently managed offenders, with RAR requirement and only a single active sentence",
+        notes = "Accepts a Delius Username. No backward compatibility guaranteed - intended for the use of the Manage a Supervision service, behaviour or responses may be modified in the future.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+        @ApiResponse(code = 404, message = "Not found", response = ErrorResponse.class),
+        @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)})
+    @GetMapping(path = "/staff/username/{username}/cases")
+    public Page<StaffCaseloadEntry> getCases(@ApiParam(name = "username", value = "Delius username", example = "SheliaHancockNPS", required = true)
+                                                           @NotNull
+                                                           @PathVariable(value = "username") final String username,
+                                             @PageableDefault(sort = {"secondName", "firstName"}, direction = Direction.ASC, size = Integer.MAX_VALUE) final Pageable pageable) {
+        return staffService.getOffenderCasesForUser(username, pageable);
+    }
```

Comment:
Small bit of caution: Delius *optionally* maps a Delius user and staff account. But that mapping is not always present and we have found a significant number where that linking hasn't happened.
The HDC licences project has been busy fixing this as it roles out region by region, and each region it roles out to we might find 20 or 30 staff accounts not linked.

Therefore this won't always return results as expected

Reply:
Thanks @andymarke. In these cases how is the staff id -> user mapping resolved. Is it a manual process or is there an alternative way to join tables to determine the staff_id from the username?